vignettes/social_smell_showcase.Rmd

```diff
@@ -94,7 +97,16 @@ As stated in the introduction, we need both git log and at least one communicati
 
 ## Parse Gitlog
 
+Recommended perceval version: 0.12.24
```

Comment:
This is specific to Kaiaulu, rather than just this notebook, so let's leave this one out since one PR already adds it to the README (or at least one issue I have that noted down).